xrpl4j-integration-tests/src/test/java/org/xrpl/xrpl4j/tests/environment/MainnetEnvironment.java

```diff
@@ -29,8 +29,13 @@
  */
 public class MainnetEnvironment implements XrplEnvironment {
 
-  // s1 and s2 are both powered by Clio (a full-history node operated by Ripple).
-  private final XrplClient xrplClient = new XrplClient(HttpUrl.parse("https://s1.ripple.com:51234"));
+  // FIXME: At the time of writing this, Clio is powering some traffic to s1 and s2. Clio behaves differently
+  //  than reporting mode servers/rippled p2p servers in responses to account_tx RPC calls with
+  //  "ledger_index" = "validated", which breaks AccountTransactionsIT. xrplcluster.com is not powered by Clio, so
```

Comment:
Let's do the following:

1. Break `TestnetEnvironment` into two: `ReportingTestnetEnvironment` (points to https://s.altnet.rippletest.net:51234) and `ClioTestnetEnvironment` (points to https://clio.altnet.rippletest.net:51234).
2. In CI, run all ITs against both environments.
3. If the ClioTestnet environment fails, we'll log a bug in Clio, but merge xrp4lj PRs (i.e., clio tests are not required to pass).